### DISCUSSION TITLE
alpine3.16 runtime prints stack trace but alpine3.17 doesn't

### DISCUSSION THREAD

**Discussion by andysbolton** [dotnet/dotnet-docker#4379] [Q&A] [answered]
Hi, I can't tell if this is related to a bug or my own misconfiguration, but I would love another opinion.

I've noticed that changing my dotnet runtime from `mcr.microsoft.com/dotnet/aspnet:6.0.13-alpine3.16` to `mcr.microsoft.com/dotnet/aspnet:6.0.13-alpine3.17` results in a stack trace no longer printing to standard out when I run a container with a non-zero exit code. Please see [this repo](https://github.com/andysbolton/alpine317segfault) for an example.

Given a simple program targeting `mcr.microsoft.com/dotnet/aspnet:6.0.13-alpine3.16` that throws an exception, a stack trace is printed:
![image](https://user-images.githubusercontent.com/5191998/216201945-0ba4ff6f-dbb9-4ad1-b563-0178afba67c6.png)
But it will not be printed when targeting `mcr.microsoft.com/dotnet/aspnet:6.0.13-alpine3.17`:
![image](https://user-images.githubusercontent.com/5191998/216200481-e0329e96-3ee7-443a-9c13-8857a62a76cb.png)

Furthermore, if I `exec` into a container with the `mcr.microsoft.com/dotnet/aspnet:6.0.13-alpine3.17` runtime and run `dotnet alpine317segfault.dll`, I receive a segmentation fault:
![image](https://user-images.githubusercontent.com/5191998/216201036-5f7a3f7b-4f1c-4c5d-9874-8e97aad18a5d.png)

```
docker -v                                                                                                         
Docker version 20.10.22, build 3a2c30b
```

Is this something I should take up with alpine? Is there something else obvious that I'm missing?

Thanks!

**Comment by mthalman** [comment, marked as answer]
It looks like you're running into this: https://github.com/dotnet/runtime/issues/81218
